### DISCUSSION TITLE
Bypass for enchantment costs

### DISCUSSION THREAD

**Discussion by TheoRetisch1** [Zrips/CMI#2824] [Feature Requests]
If someone wants to enchant an item in survival mode or wants to combine two items and the process costs more than 40 levels this is not possible. In the anvil the message too expensive appears.

It would be great if you could bypass this process with a permission just like the repair costs: 
cmi.anvil.enchantment.nolimits

The process is part of the daily business on many servers and would benefit a broad majority.

**Comment by impmallet** [comment]
I agree, this would be very useful.

**Comment by TheoRetisch1** [comment]
I would still love to see this feature :)

**Comment by Isaiah1174** [comment]
There's already a permission "cmi.anvil.nolimits" but I don't believe it's working at the moment so I'm about to submit a bug report on it

**Comment by TheoRetisch1** [comment]
The permission cmi.anvil.nolimit is working, it allows a player to repair items without level limitations. But I want to enable players to enchant without level limitations. I would love to see the feature :)
